components/MultiSelect.tsx

```diff
@@ -10,18 +10,23 @@ interface Option {
 interface MultiSelectProps {
   name: string;
   options: Option[];
-  placeholder: string;
+  // eslint-disable-next-line react/require-default-props
+  validate?: (value: string[]) => string | undefined;
 }
 
 const MultiSelect = ({
   name,
   options,
-  placeholder,
+  validate = undefined,
```

Comment:
`?` implies prop can be undefined -- the eslint rule you escaped shouldn't require us to set this default prop value?

Reply:
Fixed, thanks! Might've been because I had it as `(value: string[]) => string | undefined`